src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -439,6 +439,27 @@ def get_encryption_config(self, daemon_name=None, owner=None):
         return CephService.get_encryption_config(daemon_name)
 
 
+@UIRouter('/rgw/bucket', Scope.RGW)
+class RgwBucketUi(RgwBucket):
+    @Endpoint('GET')
+    @ReadPermission
+    # pylint: disable=W0613
+    def buckets_and_users_count(self, daemon_name=None):
```

Comment:
I'm still wondering, if this a string, shouldn´t this be like this?

```suggestion
    def buckets_and_users_count(self, daemon_name: str = ""):
```

Reply:
Since daemon_name is defaulted as None everywhere throughout the code..so to be consistent with this behavior its defaulted to None here as well